config/environments/test.rb

```diff
@@ -50,6 +50,14 @@
   config.assets.compile = true
   config.assets.digest = false
 
+  config.active_record.verbose_query_logs = true
+
+  config.after_initialize do
+    ActiveRecord::LogSubscriber.backtrace_cleaner.add_silencer do |line|
+      line =~ %r{lib/freedom_patches}
```

Comment:
This is required so that the backtrace doesn't point to our freedom patches and instead point to the application code.